testing/MLDB-1155_csv_line_endings.py

```diff
@@ -199,7 +197,7 @@
     assert json.loads(result['response'])[1][2] == 1
 
 
-    # CSV importation should accept a blank trailing line with DOS
+    # CSV importation should accept a blank trailing line with DOS if ignoreBadLines is set to true
```

Comment:
This is not easily fixed.  The callers of forEachLineBlock have no idea if a line is the last one and forEachLineBlock should not drop lines.  Not sure what to do here.